crates/cxx-qt-gen/src/syntax/qtfile.rs

```diff
@@ -12,7 +12,7 @@ use syn::{AttrStyle, Attribute, Result};
 /// Representation of a CxxQtFile as Syn items
 pub struct CxxQtFile {
     /// A vector of [syn::Attribute] in the file
-    pub attrs: Vec<Attribute>,
+    pub attrs: Vec<Attribute>, // TODO: Should these be checked at this point?
```

Comment:
same i think, this is just a helper for the shape of the CXX-Qt syntax in a syn-like struct that then parser uses

Reply:
Removed comment